js/src/button.js

```diff
@@ -143,10 +139,7 @@ class Button {
 EventHandler.on(document, EVENT_CLICK_DATA_API, SELECTOR_DATA_TOGGLE_CARROT, event => {
   event.preventDefault()
 
-  let button = event.target
-  if (!button.classList.contains(CLASS_NAME_BUTTON)) {
```

Comment:
Is this intentional?

Reply:
Yes, `.closest()` returns the element itself if it contains the class (see https://developer.mozilla.org/en-US/docs/Web/API/Element/closest)